src/Akeneo/Connectivity/Connection/front/src/connect/components/ConnectedApp/ConnectedAppContainer.tsx

```diff
@@ -21,6 +20,7 @@ import isGrantedOnProduct from '../../is-granted-on-product';
 import isGrantedOnCatalog from '../../is-granted-on-catalog';
 import {CatalogList} from '@akeneo-pim-community/catalogs';
 import styled from 'styled-components';
+import {OpenAppButton} from "./OpenAppButton";
```

Comment:
If it does not warn at the CI, we should add lint rule.
```suggestion
import {OpenAppButton} from './OpenAppButton';
```